sp/src/game/server/hl2/ai_allymanager.cpp

```diff
@@ -164,7 +164,7 @@ void CAI_AllyManager::CountAllies( int *pTotal, int *pMedics )
 
 #ifdef EZ
 			// Blixibon - The soldiers' "IsCommandable" function only checks the spawnflag, so it's fine here
-			if( FClassnameIs( ppAIs[i], "npc_combine_s" ) ) 
+			if( FClassnameIs( ppAIs[i], "npc_combine_s" ) || FClassnameIs( ppAIs[i], "npc_husk_soldier" ) ) 
```

Comment:
What about other derivatives of the Combine soldier like Bad Cop, Clone Cop, and the Combine assassins?

Reply:
I've actually never thought about that 🤔

We should add them in the future, but I think I'd rather sort that out in another PR unless you think we should hold off on this one otherwise.

Reply:
I think we should hold off on this one just because I think it's unlikely we'll return to this issue and I could see this being needed for assassins at least. Some poor mapper is going to need it for Clone Cops or Bad Cops too. Maybe it should check if the entity is descended from the base Combine class?